lidar:
  lidar_topic: /ouster/points
  frame_id: os_lidar

camera:
  image_topic: /image_raw
  projected_topic: /point_projected_image
  image_size:
    width: 640
    height: 360
  frame_id: default_cam

chessboard:
  pattern_size:
    rows: 8
    columns: 9
  square_size_meters: 0.07

general:
  config_folder: /home/wonsuk1025/kai_ws/src/ros2_camera_lidar_fusion/config
  data_folder: /home/wonsuk1025/kai_ws/src/ros2_camera_lidar_fusion/data
  correspondence_file: ros2_camera_lidar_fusion_correspondences.txt
  camera_intrinsic_calibration: camera_intrinsic_calibration.yaml
  camera_extrinsic_calibration: camera_extrinsic_calibration.yaml
  slop: 0.1 
  max_file_saved: 25
  keyboard_listener: True
  get_intrinsics: True
  get_extrinsics: True
